commonFunctions.py

```diff
@@ -194,25 +195,44 @@ def dropout(self, value):
     else:
       raise TypeError("dropout must be a double")
 
+  @property
+  def l2(self):
+    """The 'l2' property"""
+    if self._verbose:
+      print "Getter of 'l2' called"
+    return self._l2
+  @l2.setter
+  def l2(self, value):
+    """Setter of the 'l2' property"""
+    if isinstance(value, (int, long, float)):
+      if value < 0 or value >= 1:
+        raise ValueError("Fraction must be between 0 and 1")
```

Comment:
Fix this part. This check came from copying from another line. For the l2, while small values are the ones preferred, there is in fact no upper limit on the parameter. Change the check to only look at the lower bound, i.e. l2 should be positive